src/components/Button/index.tsx

```diff
@@ -40,16 +44,20 @@ const Button = React.forwardRef<HTMLButtonElement, Props>(function Button(
         disabled = false,
         tooltip,
         onClick,
+        theme,
```

Comment:
Possibly dumb question given that I just started to look at this diff and may lack context: Shouldn't the theme be global and maybe provided via a context? It does not seem scalable to be having to pass it to every component.

Reply:
I think the idea here is: we read the `theme` from a context but can override on a per component-instance basis by providing a different theme in props

Reply:
Yup that's it! Even though we have a global theme via a ThemeProvider (not yet in this PR), we might need a selective override for a single component.

Reply:
Adding to this, if we use a real context provider we can use additional nested providers rather than using a prop. Example: https://evergreen.segment.com/get-started/theming